src/ext-test/resources/vehicleparking/bikely/bikely.json

```diff
@@ -2622,6 +2719,7 @@
     {
       "name": "KLP EL-pool",
       "availableParkingSpots": 1,
+      "totalParkingSpots": 0,
```

Comment:
What does it mean that `totalParkingSpots` is zero for most of the places?

Reply:
It means that their API is buggy. I've already reported it and it will be fixed.

Reply:
Now that I think about it, we may want to skip the capacity if it's null. WDYT?